### ISSUE TITLE
Quzi and test automation

### ISSUE THREAD

**Issue by boolinb** [GSRHackZ/EdgyBot_Edgenuity_Bot#128]
Will Automation ever come back out for Quizzes Practices or Tests or is that never coming back again.

**Comment by GSRHackZ** [comment]
Yes, within these two weeks it will be added again, most likely this weekend,
